tensorflow_addons/metrics/multilabel_metrics.py

```diff
@@ -0,0 +1,235 @@
+import tensorflow as tf
+from tensorflow.keras.metrics import Metric
+
+class MultiLabelMacroRecall(Metric):
+    """Computes the Macro-averaged Recall of the given tensors."""
+
+    def __init__(
+        self,
+        name:str="multi_label_macro_recall",
+        threshold:float=0.5,
+        from_logits:bool=True,
+        activation:str="sigmoid",
+        **kwargs
+    ):
+        """Creates a `MultiLabelMacroRecall` instance.
+
+        Args:
+            name (str): (Optional) string name of the metric instance. Defaults to
+                `"multi_label_macro_recall"`.
+            threshold (float): (Optional) float value to use to binarize labels and
+                predictions. Values greater than `threshold` will set to `1`.
+                Defaults to `0.5`.
+            from_logits (bool): (Optional) boolean value to specifiy whether or not
+                `y_pred` are logits or have not yet been transformed via a final
+                activation function. If `from_logits` is `False`, `activation`
+                will be applied to `y_pred` prior to metric calculation. Defaults
+                to `True`.
+            activation (str): (Optional) string value of the activation function
+                to apply to `y_pred` if `from_logits=False`. Defaults to `"sigmoid"`
+                . Options include `["sigmoid"]`.
```

Comment:
Please follow the doc in core TF if they have same semantics.

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/metrics.py#L1182-L1195

Reply:
there is inconsistency there, but I switched from Python's PEP and Googles official styleguide to what I believe is the proper from the main repo's metrics e.g. no type annotations, and instead of specifically saying default saying "if option [default] is selected then..."